src/gt4py/next/program_processors/runners/gtfn_cpu.py

```diff
@@ -134,6 +134,9 @@ def compilation_hash(otf_closure: stages.ProgramCall) -> int:
 ](
     name="run_gtfn_with_temporaries",
     otf_workflow=run_gtfn.otf_workflow.replace(
-        translation=run_gtfn.otf_workflow.translation.replace(lift_mode=LiftMode.FORCE_TEMPORARIES),
+        translation=run_gtfn.otf_workflow.translation.replace(
```

Comment:
Temporary extraction without the heuristics is not useful in almost all cases. I just switched to using the heuristics by default.